Tests/TestCodeGenConfigurations/SchemaCustomization/Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.7
+
+import PackageDescription
+
+let package = Package(
+  name: "TestApp",
+  platforms: [
+    .iOS(.v12),
+    .macOS(.v10_15),
+    .tvOS(.v12),
+    .watchOS(.v5),
+  ],
+  products: [
+    .library(name: "TestApp", targets: ["TestApp"]),
+  ],
+  dependencies: [
+    .package(name: "apollo-ios", path: "../../../apollo-ios"),
+    .package(name: "AnimalKingdomAPI", path: "./AnimalKingdomAPI")
+  ],
+  targets: [
+    .target(
+      name: "TestApp",
+      dependencies: [
+        .product(name: "Apollo", package: "apollo-ios"),
+        .product(name: "AnimalKingdomAPI", package: "AnimalKingdomAPI")
+      ]
```

Comment:
Since we are adding the `ExistentialAny` requirement in #379:

```suggestion
      name: "TestApp",
      dependencies: [
        .product(name: "Apollo", package: "apollo-ios"),
        .product(name: "AnimalKingdomAPI", package: "AnimalKingdomAPI")
      ],
      swiftSettings: [.enableUpcomingFeature("ExistentialAny")]
```